packages/fluttium_driver/pubspec.yaml

```diff
@@ -11,7 +11,8 @@ environment:
 
 dependencies:
   equatable: ^2.0.5
-  fluttium_interfaces: ^0.1.0
+  fluttium_interfaces:
+    path: ../fluttium_interfaces
```

Comment:
Needs to be reverted

Reply:
The [pubspec_overrides.yaml](https://github.com/wolfenrain/fluttium/blob/main/packages/fluttium_cli/pubspec_overrides.yaml) already does this for u 😉